Link to the forum instead of Slack (#395)
exercism/docs: building/tooling/analyzers/creating-from-scratch.md

3. Open an issue at [exercism/exercism][exercism-repo] introducing yourself and telling us which language you'd like to create a Analyzer for.
4. Once an Analyzer repo has been created, use [the Analyzer interface document](/docs/building/tooling/analyzers/interface) to help guide your implementation.

> We have an incredibly friendly and supportive community who will be happy to help you as you work through this! If you get stuck, please speak to us on [the Exercism forum][forum] or create new issues at [exercism/exercism][exercism-repo] as needed 🙂

Reviewer: ```suggestion
We have an incredibly friendly and supportive community who will be happy to help you as you work through this! If you get stuck, please start a new topic on [the Exercism forum][forum] or create new issues at [exercism/exercism][exercism-repo] as needed 🙂
```

> [forum]: https://forum.exercism.org/c/exercism/building-exercism/125

Reviewer 2: Maybe make this `[building-exercism]` or similar so that the `forum` link label has a consistent destination across all docs?
